cluster/manifests/skipper/routegroup-crd.yaml

```diff
@@ -90,56 +123,69 @@ spec:
                   type: object
                 type: array
               defaultBackends:
-                description: DefaultBackends is a list of default backends defined if no explicit backend is defined for a route.
+                description: DefaultBackends is a list of default backends defined
+                  if no explicit backend is defined for a route
                 items:
                   properties:
                     backendName:
-                      description: BackendName references the skipperBackend by name
+                      description: BackendName references backend by name
                       type: string
                     weight:
-                      description: Weight defines the traffic weight, if there are 2 or more default backends
+                      description: Weight defines a portion of traffic for the referenced
+                        backend. It equals to weight divided by the sum of all backend
+                        weights. When all references have zero (or unspecified) weight
+                        then traffic is split equally between them.
                       minimum: 0
                       type: integer
                   required:
                   - backendName
                   type: object
                 type: array
               hosts:
-                description: List of hostnames for the RouteGroup.
+                description: List of hostnames for the RouteGroup
                 items:
 {{ if eq .ConfigItems.skipper_routegroup_enable_hostport "true"}}
```

Comment:
Maybe we should get rid of `skipper_routegroup_enable_hostport` first.

Reply:
why?

Reply:
https://github.com/zalando-incubator/kubernetes-on-aws/pull/6064